lib/commands.js

```diff
@@ -1,4 +0,0 @@
-module.exports = [
```

Comment:
I'm not sure that it needs to be maintained here, but I do think a listing of the possible commands (with descriptions and maybe a function to test whether a message should trigger a given command) would be useful. Since we're adding command-oriented functionality in this PR (`@pr-police me`), I think another essential command is `help`, which should probably be run if `@pr-police help` or maybe even just an empty `@pr-police` message and would print the possible commands. It would be even better if that help command were visible just to the user that sent it (not sure how that is actually done in slack).